react/src/sagas/Header.js

```diff
@@ -22,8 +22,10 @@ export function* requestLogout () {
             yield put(resp);
         }
     } catch(error) {
-        console.log('Error performing logout: ' + error.toString());
-        yield put(postLogoutFail(error.toString()));
+        if (notIE11Unauthn(error) === true) {
+            console.log('Error performing logout: ' + error.toString());
+            yield put(postLogoutFail(error.toString()));
```

Comment:
Is this a special case or just forgotten?

Reply:
Forgotten! And weird, since I grepped for notIE11Unauthn...